### PR TITLE
LPD-34182 Prevent try to access objectEntryLocalService when the object definition's storage type is different from Default

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPD-34182

**NOTE:** Caught by the test: 
`LocalFile.ObjectSalesforce#AssertCRUDWithCreatedCustomObject`

### CODE REVIEW

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContextImpl.java [hunk 1]
```diff
@@ -190,21 +190,17 @@ public String getBackURL() throws PortalException {
 			backURL = String.valueOf(liferayPortletResponse.createRenderURL());
 		}
 
-		if (objectEntry == null) {
+		ObjectDefinition objectDefinition = getObjectDefinition1();
+
+		if ((objectEntry == null) || !objectDefinition.isDefaultStorageType() ||
+			!objectDefinition.isRootDescendantNode()) {
+
```
Comment (carolmariaabb) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContextImpl.java:197]: @p-albuquerque you can optimize the code:

```
ObjectDefinition objectDefinition = getObjectDefinition1();

if (!objectDefinition.isDefaultStorageType() || !objectDefinition.isRootDescendantNode()) {
    return backURL;
}

ObjectEntry objectEntry = _getObjectEntry();

if (objectEntry == null) {
   return backURL;
}
```

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by p-albuquerque** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward:force

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**